onthego/project/sqlqueries.py

```diff
@@ -0,0 +1,19 @@
+from .models import db
+
+def logininsert(model,**kwargs):
```

Comment:
Learn more about clean code practices. Naming does not meet the coding standard.